source/index.html.md.erb

```diff
@@ -11125,16 +11167,21 @@ A Sheet can exist in a user's **Sheets** folder ([Home](#home)), in a [Folder](#
 
      |     |     |
 -----|-----|-----|
-**readOnlyLiteEnabled** | Boolean | If `true`, a lightweight version of the sheet is published without row attachments or discussions.
-**readOnlyFullEnabled** | Boolean | If `true`, a rich version of the sheet is published with the ability to download row attachments and discussions.
-**readWriteEnabled** | Boolean | If `true`, a rich version of the sheet is published with the ability to edit cells and manage attachments and discussions.
 **icalEnabled** | Boolean | If `true`, a webcal is available for the calendar in the sheet.
+**icalUrl** | string | URL for 'iCal' view of the published sheet<br/>Only returned in a response if **icalEnabled** = true.
 **readOnlyFullAccessibleBy** | string | Flag to indicate who can access the 'Read-Only Full' view of the published sheet:<ul><li>If "ALL", it is available to anyone who has the link.</li><li>If "ORG", it is available only to members of the sheet owner's Smartsheet organization.</li></ul>Only returned in a response if **readOnlyFullEnabled** = true.
-**readWriteAccessibleBy** | string | Flag to indicate who can access the 'Edit by Anyone' view of the published sheet:<ul><li>If "ALL", it is available to anyone who has the link.</li><li>If "ORG", it is available only to members of the sheet owner's Smartsheet organization.</li></ul>Only returned in a response if **readWriteEnabled** = true.
-**readOnlyLiteUrl** | string | URL for 'Read-Only HTML' view of the published sheet<br/>Only returned in a response if **readOnlyLiteEnabled** = true.
+**readOnlyFullDefaultView** | enum | Indicates which view the user has set for a read-only, default view of the published sheet. 0 or null = GRID, 1 = CARD, and 2 = CALENDAR.
+**readOnlyFullEnabled** | Boolean | If `true`, a rich version of the sheet is published with the ability to download row attachments and discussions.
+**readOnlyFullShowToolbar** | Boolean | If `true`, shows the left nav toolbar.
 **readOnlyFullUrl** | string | URL for 'Read-Only Full' view of the published sheet<br/>Only returned in a response if **readOnlyFullEnabled** = true.
+**readOnlyLiteEnabled** | Boolean | If `true`, a lightweight version of the sheet is published without row attachments or discussions.
+**readOnlyLiteSSslUrl** | string | If `true`, returns an SSL link.
```

Comment:
This is an URL - never 'true'. Returned if the the sheet is published to an SSL url

Reply:
Yeah, you're right. I don't see how a string can also be a Boolean. The spec specifically mentions being able to set it to false so that nothing will be returned, but it also says it's a string. So either it's a Boolean or it's a string. It's Proj-0970 if you want to help me decipher/interpret.